packages/testing/src/fs.ts

```diff
@@ -0,0 +1,14 @@
+import { mkdir } from 'fs/promises'
+import { platform } from 'process'
+
+import { execa } from 'execa'
+
+export const unzipFile = async function (path: string, dest: string): Promise<void> {
```

Comment:
I've extracted this function from `@netlify/zip-it-and-ship-it` so it can be used by any package in the monorepo.

Reply:
I'd like to update `@netlify/zip-it-and-ship-it` to pull the function from `@netlify/testing`, but sadly I'm getting a circular dependency error. For now, I'm keeping the function in both places.